### ISSUE TITLE
Blank popup when nothing matched with search term

### ISSUE THREAD

**Issue by yueim** [p0deje/Maccy#571]
Fistly, thanks so much for sharing this great app.

If checked `Avoid taking application focus` and nothing matched search term, the popup window will be blank, shown as below:

![maccy-blank-popup-window](https://github.com/p0deje/Maccy/assets/275233/221db54f-d2c6-457d-84ac-f21473658722)

Screen captured by LICEcap.

**Comment by p0deje** [comment]
I can't reproduce the issue regardless of whether the focus is enabled or not. Maybe this is something specific to OS version? Which one do you use?

**Comment by yueim** [comment]
MacBook Air wiht Apple Silicon
macOS 13.4.1 (22F82)
Maccy 0.26.5(Universal)

Now (after rebooted last night), everything works like a charming.

Maybe it's only can be reproduced when a fresh installation?
1. It's a fresh macOS.
2. Never installed Maccy.
3. Install Maccy, then open it, first time need to allow some Accessibility permissions.
4. Then do the search test.

I'll close this issue.
Thanks again for this great app.

**Comment by yueim** [comment]
Still exists.

![maccy-blank-popup-window](https://github.com/p0deje/Maccy/assets/275233/b3353e2b-e504-4a56-916d-8357bede484a)

**Comment by p0deje** [comment]
Can you please confirm you don't run 2 instances of Maccy at the same time? It acts and looks weird in general.

**Comment by yueim** [comment]
1. Only one instance.
2. Reproduced only once.

![maccy-blank-popup-window-02](https://github.com/p0deje/Maccy/assets/275233/1f8dd3ce-fa5b-450f-af74-5347ce71a582)

I think thiis is a very small bug and the priority is low.

Thanks.

**Comment by yueim** [comment]
1. Confirmed that there's only one instance is running.
2. Today I can only reproduce the blank window one time.
3. I think it's just a low priority bug.

![maccy-blank-popup-window-02](https://github.com/p0deje/Maccy/assets/275233/1f8dd3ce-fa5b-450f-af74-5347ce71a582)
